service/controller/v13/adapter/recordsets.go

```diff
@@ -1,32 +1,69 @@
 package adapter
 
+import (
+	"github.com/giantswarm/aws-operator/service/controller/v13/cloudformation"
+	"github.com/giantswarm/aws-operator/service/controller/v13/key"
+	"github.com/giantswarm/microerror"
+)
+
 // The template related to this adapter can be found in the following import.
 //
 //     github.com/giantswarm/aws-operator/service/controller/v13/templates/cloudformation/guest/recordsets.go
 //
 
 type recordSetsAdapter struct {
-	APIELBHostedZones         string
-	APIELBDomain              string
-	EtcdELBHostedZones        string
-	EtcdELBDomain             string
-	IngressELBDNS             string
-	IngressELBHostedZones     string
-	IngressELBAliasHostedZone string
-	IngressELBDomain          string
-	IngressWildcardELBDomain  string
-	Route53Enabled            bool
+	BaseDomain                 string
+	GuestHostedZoneNameServers string
```

Comment:
When there are multiple name servers, would `[]string` make more sense?

Reply:
CF allows only string outputs, this string is taken directly from the output I don't want to manipulate in the code. I also wouldn't be surprised if array inputs aren't allowed, but I don't have energy to check that.